extension.json

```diff
@@ -0,0 +1,160 @@
+{
+  "name": "HummingBird",
```

Comment:
I guess we need a more descriptive name if we want to publish.

Should be okay right now.